metastore/src/java/org/apache/hadoop/hive/metastore/SerDeStorageSchemaReader.java

```diff
@@ -45,7 +45,7 @@
       }
 
       Deserializer s = HiveMetaStoreUtils.getDeserializer(conf, tbl, null, false);
-      return HiveMetaStoreUtils.getFieldsFromDeserializer(tbl.getTableName(), s);
+      return HiveMetaStoreUtils.getFieldsFromDeserializer(tbl.getTableName(), s, null);
```

Comment:
Would it make sense to keep the original method `HiveMetaStoreUtils.getFieldsFromDeserializer()` which calls the new overloaded method with defaultComment=null?

Reply:
Changed the whole approach since we do not have enough info here to decide on the default comment